### PR TITLE
Move jwt/login to microservice

### PR DESCRIPTION
**Pull request by MishaHMK**
## SUMMARY
Reused code from [my previous pr](https://github.com/project-studying-dotnet/Streetcode-Server-September/pull/182/files#diff-bb0d97b7e937337f22e15f23e49a48691869e39ae6baab7cfcf66de048f8ea9e), with some minor changes (validator doesn't have a lot of elements of main API)

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=50%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streedcode.Identity/Program.cs [hunk 1]
```diff
@@ -34,6 +37,8 @@
     app.UseSwaggerUI();
 }
 
+await app.SeedDataAsync();
+
 await app.ApplyMigrations();
```
Comment (jimmy2jk) [inline Streetcode/Streedcode.Identity/Program.cs:42]: SeedDataAsync must be called adter ApplyMigrations

File: Streetcode/Streedcode.Identity/Services/Realizations/JwtService.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        public async Task<string> Create(ApplicationUser user)
+        {
+            string secretKey = _secret;
+            var securityKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(secretKey));
+
+            var credentials = new SigningCredentials(securityKey, SecurityAlgorithms.HmacSha256);
+
+            var claims = new List<Claim> {
+                new Claim(ClaimsIdentity.DefaultRoleClaimType, user.Id.ToString())
+            };
```
Comment (jimmy2jk) [inline Streetcode/Streedcode.Identity/Services/Realizations/JwtService.cs:45]: Fix formatting. Opening curly braces should be pleced on a new line.

File: Streetcode/Streedcode.Identity/Services/Realizations/JwtService.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                 Expires = DateTime.UtcNow.AddMinutes(_expiration),
+                 SigningCredentials = credentials,
+                 Issuer = _issuer,
+                 Audience = _audience
+             };
+
+             var handler = new JsonWebTokenHandler();
+
+             string token = handler.CreateToken(tokenDescriptor)??
+                   throw new InvalidOperationException("Token generation failed"); ;
```
Comment (jimmy2jk) [inline Streetcode/Streedcode.Identity/Services/Realizations/JwtService.cs:65]: one extra `;`

**Review by jimmy2jk — CHANGES_REQUESTED** [review]
Fix order of SeedData and ApplyMigrations. Also little changes to formatting required

**Review by jimmy2jk — APPROVED** [review]

**Review by KarmazinN — APPROVED** [review]

**Pull request merged**